src/components/NewsFeed.js

```diff
@@ -119,17 +125,18 @@ const NewsFeed = ({ news, isLoading, isMoreAvailable, onLoadMore }) => {
   );
 };
 
-const NewsList = ({ news }) => {
+const NewsList = ({ news, lastSeenAt }) => {
   const newsListElement = news.map((item) => {
-    return <NewsItem key={item.id} newsItem={item} />;
+    return <NewsItem key={item.id} lastSeenAt={lastSeenAt} newsItem={item} />;
   });
 
   return <Box>{newsListElement}</Box>;
 };
 
-const NewsItem = ({ newsItem }) => {
+const NewsItem = ({ newsItem, lastSeenAt }) => {
   const classes = useStyles();
-  const [isExpanded, setIsExpanded] = useState(false);
+
+  const [isExpanded, setIsExpanded] = useState(newsItem.createdAt > lastSeenAt);
```

Comment:
I think we should probably use date here and have it in a format that can be parsed into display date and be used to make this check. However I will leave this for now and we can figure out the rest after we have the backend